### PR TITLE
feat: add Turkish lang file

### PR DESCRIPTION
**Pull request by mcsaygili**
Added Turkish language support.

### CODE REVIEW

File: src/Language/tr/Auth.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CodeIgniter\Shield\Language\en;
```
Comment (kenjis) [inline src/Language/tr/Auth.php:5]:
```suggestion
namespace CodeIgniter\Shield\Language\tr;
```

Comment (datamweb) [reply src/Language/tr/Auth.php]: Done.

**Review by datamweb — APPROVED** [review]
Hello, thank you.🥀
müteşekkir.

**Comment by mcsaygili** [comment]
I thank you for such a beautiful framework.

**Review by kenjis — APPROVED** [review]

**Review by MGatner — APPROVED** [review]
Very nice! Thank you for your contribution.

**Comment by MGatner** [comment]
We should exclude language files from code coverage.

**Pull request merged**